mlflow/utils/cli_args.py

```diff
@@ -114,6 +120,22 @@ def _resolve_env_manager(ctx, _, env_manager):
 """,
 )
 
+ENV_MANAGER_PROJECTS = _create_env_manager_option(
+    help_string="""
+If specified, create an environment for MLproject using the specified
+environment manager. The following values are supported:
+
+\b
+- local: use the local environment
+- conda: use conda
```

Comment:
Can we document virtualenv first, ahead of conda?

Reply:
Fixed!